etna/metrics/__init__.py

```diff
@@ -1,13 +1,15 @@
-from sklearn.metrics import mean_absolute_error as mae
-from sklearn.metrics import mean_squared_error as mse
-from sklearn.metrics import mean_squared_log_error as msle
-from sklearn.metrics import median_absolute_error as medae
-from sklearn.metrics import r2_score
-
 from etna.metrics.base import Metric
 from etna.metrics.base import MetricAggregationMode
+from etna.metrics.base import MetricFunctionSignature
```

Comment:
Why do you thing is useful to add these classes to init?

Reply:
Ok, let's remove them from init. But we shouldn't forget to add them into our documentation api reference after the update.